zerver/lib/emoji.py

```diff
@@ -46,7 +46,7 @@ def translate_emoticons(text: Text) -> Text:
     codepoint_to_name = ujson.load(fp)
 
 def emoji_name_to_emoji_code(realm: Realm, emoji_name: Text) -> Tuple[Text, Text]:
-    realm_emojis = realm.get_emoji()
+    realm_emojis = realm.get_active_emoji()
```

Comment:
Should this change go with removing the "deactivated" check 2 lines down?  Could probably be split out as a prep commit.